core/src/main/kotlin/fr/sncf/osrd/stdcm/graph/STDCMNode.kt

```diff
@@ -32,10 +38,13 @@ data class STDCMNode(
 ) : Comparable<STDCMNode> {
 
     /**
-     * Defines the estimated better path between 2 nodes, in terms of total run time, then departure
-     * time, then number of reached targets. If the result is negative, the current node has a
-     * better path, and should be explored first. This method allows us to order the nodes in a
-     * priority queue, from the best path to the worst path. We then explore them in that order.
+     * Defines the estimated better path between 2 nodes, in the following priority
+     * - lowest total run time
+     * - earliest departure time
+     * - closest planned arrival time, using the current then the last planned timing data
+     * - highest number of reached targets If the result is negative, the current node has a better
```

Comment:
Missing a line break there ;)

Reply:
Core format keeps putting it back it seems x) I agree though.